src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -85,6 +78,19 @@ def initialize_resubmission(request_args, acceptedArgs, reqmgr_db_service):
         if arg not in request_args and arg in acceptedArgs:
             request_args[arg] = parent_args[arg]
 
+def initialize_clone(request_args, reqmgr_db_service):
+    """
+    Initialize a Clone arguments by inheriting and overwriting argument from OriginalRequest
+    """
+    requests = reqmgr_db_service.getRequestByNames(request_args["OriginalRequestName"])
+    clone_args = requests.values()[0]
+    # TODO: need to validate new overwrite request_args here since it will skip the clone_args validtaiton
+    for prop in request_args:
```

Comment:
You mean only some part of inner dict not the whole thing. I wonder whether we should allow removing some params as well.